### PR TITLE
Create the OSCAP DBus module

### PR DESCRIPTION
**Pull request by poncovka**
The current support for Anaconda add-ons will be removed.
Use the support for Anaconda DBus add-ons instead.

**Depends on:** https://github.com/OpenSCAP/oscap-anaconda-addon/pull/133, https://github.com/OpenSCAP/oscap-anaconda-addon/pull/135, https://github.com/OpenSCAP/oscap-anaconda-addon/pull/137, https://github.com/rhinstaller/anaconda/pull/3147

### CODE REVIEW

File: tests/test_installation.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+import logging
+
+from org_fedora_oscap.service import installation
+from org_fedora_oscap.structures import PolicyData
+
+# FIXME: Extend the tests to test all paths of the installation tasks.
```
Comment (poncovka) [inline tests/test_installation.py:23]: The installation tasks are quite complex and I am not so familiar with the pytest library, so I will leave the extension of the tests up to you.

File: org_fedora_oscap/service/kickstart.py [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+
+        ret += "\n%end\n\n"
+        return ret
+
+
+class OSCAPKickstartSpecification(KickstartSpecification):
+    """The kickstart specification of the OSCAP service."""
+
+    addons = {
+        "org_fedora_oscap": OSCAPKickstartData
```
Comment (poncovka) [inline org_fedora_oscap/service/kickstart.py:252]: Btw. this is the place where you can "register" another add-on name. You will have to update the `process_kickstart` and `setup_kickstart` methods to make it work with the rest of the module, but that should be it.

**Comment by poncovka** [comment]
The code is complete, but it needs more testing after merging the dependencies.

**Comment by matejak** [comment]
Merging, there are some issues left to address, but they will be easier to work on in small subsequent PRs, and this PR improves the state considerably.

**Pull request merged**